### PR TITLE
add ng2-bootstrap v1.0.1-beta.2

### PR DESCRIPTION
**Pull request by valorkin**
git repo url: [https://github.com/valor-software/ng2-bootstrap](https://github.com/valor-software/ng2-bootstrap)
Watch 40
Star 272
Fork 51
this PR for  #6805

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
- [x] Has valid tags for each versions (for git auto-update)
- [ ] Auto-update setup
- [ ] Auto-update target/source is valid.
- [ ] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/ng2-bootstrap/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "type": "git",
+    "url": "git+ssh://git@github.com/valor-software/ng2-bootstrap.git"
+  },
+  "homepage": "https://github.com/valor-software/ng2-bootstrap#readme",
+  "npmName": "ng2-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/ng2-bootstrap/package.json]: Should not use `"**/*"` here, this is used to include all the files in its npm package.

Looks like we can set basePath to bundles, and change `"**/*"` to `"**/ng2-bootstrap*.+(js|css)"` ?

**Comment by valorkin** [comment]
@maruilian11 updated accordingly to @PeterDaveHello comment, please check

**Comment by PeterDaveHello** [comment]
Thanks @valorkin

**Comment by valorkin** [comment]
@PeterDaveHello how can I monitor when it will be available at cdnjs?
https://cdnjs.com/

**Comment by valorkin** [comment]
it is already there, thanks:)

**Comment by PeterDaveHello** [comment]
@valorkin it takes 15~30 mins in usual, you can search in on cdnjs.com, glad you saw it

File: ajax/libs/ng2-bootstrap/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "type": "git",
+    "url": "git+ssh://git@github.com/valor-software/ng2-bootstrap.git"
+  },
+  "homepage": "https://github.com/valor-software/ng2-bootstrap#readme",
+  "npmName": "ng2-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "bundles",
+      "files": [
+        "**/ng2-bootstrap*.(js|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/ng2-bootstrap/package.json]: Should be `"**/ng2-bootstrap*.+(js|map)"`

**Comment by PeterDaveHello** [comment]
@valorkin if I wanna add old versions, what are the files should be included?

**Comment by valorkin** [comment]
@PeterDaveHello none, only in this version I have added system.js bundles

**Comment by valorkin** [comment]
@PeterDaveHello huge thanks for you fix

**Comment by PeterDaveHello** [comment]
no problem

**Pull request merged**